test/e2e/storage/testsuites/provisioning.go

```diff
@@ -701,7 +701,9 @@ func (t StorageClassTest) checkProvisioning(client clientset.Interface, claim *v
 				break
 			}
 		}
-		framework.ExpectEqual(found, true)
+		if !found {
+			framework.Failf("Access mode %v is not in claim's access mode", pvMode)
```

Comment:
Can you add the actual access modes here, not just the expected one?

Reply:
updated